test_crates/trait_items_hidden_from_public_api/new/src/lib.rs

```diff
@@ -1,21 +1,16 @@
-pub trait NonSealed {
-    /// Changing the bounds on an associated type is breaking,
-    /// even if the type was hidden.
-    #[doc(hidden)]
+pub trait PublicAPISealed {
+    /// Changing the bounds on a hidden type of a publically sealed trait is non breaking    #[doc(hidden)]
```

Comment:
Probably intended as:
```suggestion
    /// Changing the bounds on a hidden type of a public API sealed trait is non breaking
    #[doc(hidden)]
```